src/controllers/category.js

```
@@ -0,0 +1,13 @@
+const categoryService = require('~/services/category')
+
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)